### PR TITLE
use ERB comment to keep it out of the generated file

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

File: templates/httpd.conf.erb [hunk 1]
```diff
@@ -11,7 +11,7 @@ KeepAlive <%= @keepalive %>
 MaxKeepAliveRequests <%= @max_keepalive_requests %>
 KeepAliveTimeout <%= @keepalive_timeout %>
 LimitRequestFieldSize <%= @limitreqfieldsize %>
-# Actually >= 2.4.24, but the minor version is not provided
+<%# Actually >= 2.4.24, but the minor version is not provided -%>
```
Comment (tphoney) [inline templates/httpd.conf.erb]: Is the final '-' needed ?

Comment (mmoll) [inline templates/httpd.conf.erb]: nope, just personal style. does it need to get removed?

**Comment by tphoney** [comment]
Nope, its all good. i was just wondering if there was some ERB magic happening.

**Comment by tphoney** [comment]
THANKS @mmoll  !!!

**Pull request merged**